src/spec/integration/import.spec.coffee

```diff
@@ -891,3 +891,111 @@ describe 'Import integration test', ->
         expect(p.slug).toEqual en: @newProductSlug
         done()
       .catch (err) -> done _.prettify(err)
+
+    it 'should update a product level info based only on SKU', (done) ->
+      newProductNameUpdated = "#{@newProductName}-updated"
+      categories = []
+      for i in [1...4]
+        categories.push(        {
+          "name": {
+            "en": "Catgeory#{i}"
+          },
+          "slug": {
+            "en": "category-#{i}"
+          },
+          "externalId": "#{i}",
+        })
+
+      csv =
+      """
+        productType,name,sku,variantId,prices,categories
+        #{@productType.id},#{@newProductName},#{@newProductSku+1},1,EUR 999,1;2
+        """
+
+      TestHelpers.ensureCategories(@client, categories)
+      .then =>
+        @importer.import(csv)
+      .then (result) =>
+        expect(_.size result).toBe 1
+        expect(result[0]).toBe '[row 2] New product created.'
+
+        csv =
+        """
+          productType,sku,name.en,name.it,categories
+          #{@productType.name},#{@newProductSku+1},#{newProductNameUpdated},#{newProductNameUpdated}-it,2;3
```

Comment:
Here is updated version - 019ddde
If you provide `categories` column in CSV with an empty value, it will clear categories in Sphere. If you provide new categories - it will replace current values saved in Sphere. And if the `categories` column is not present in the CSV file, it will preserve old categories.